<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PainterPlotWidget</class>
 <widget class="QWidget" name="PainterPlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PainterPlotWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCustomPlot" name="qcustomplot_voltage" native="true"/>
   </item>
   <item>
    <widget class="QCustomPlot" name="qcustomplot_current" native="true"/>
   </item>
   <item>
    <widget class="QCustomPlot" name="qcustomplot_temperature" native="true"/>
   </item>
   <item>
    <widget class="QCustomPlot" name="qcustomplot_Soc" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
